source/elements/oneDAL/source/data_management/accessors.rst

```diff
@@ -62,28 +64,91 @@ The following table briefly explains these classes and shows which
      - Description
      - List of supported types
    * - row_accessor_
-     - Provides access to the range of dataset rows as one :term:`contiguous
+     - Provides access to the range of rows as one :term:`contiguous
        <Contiguous data>` :term:`homogeneous <Homogeneous data>` block of memory.
-     - :code:`homogen_table`, :code:`soa_table`, :code:`aos_table`,
-       :code:`csr_table`, and their builders.
+     - :code:`homogen_table`
    * - column_accessor_
      - Provides access to the range of values within a single column as one
        :term:`contiguous <Contiguous data>` :term:`homogeneous <Homogeneous
        data>` block of memory.
-     - :code:`homogen_table`, :code:`soa_table`, :code:`aos_table`,
-       :code:`csr_table`, and their builders.
-
+     - :code:`homogen_table`
+
+Usage example
+-------------
+
+Next listing provides a brief introduction of ``row_accessor`` and
+``column_accessor`` APIs and example of how to use them with ``homogen_table``.
+
+::
+
+   #include <CL/sycl.hpp>
+   #include <iostream>
+
+   #include "oneapi/dal/table/homogen.hpp"
+   #include "oneapi/dal/table/row_accessor.hpp"
+   #include "oneapi/dal/table/column_accessor.hpp"
+
+   using namespace oneapi;
+
+   int main() {
+      sycl::queue queue { sycl::default_selector() };
+
+      constexpr float host_data[] = {
+         1.0f, 1.5f, 2.0f,
+         2.1f, 3.2f, 3.7f,
+         4.0f, 4.9f, 5.0f,
+         5.2f, 6.1f, 6.2f
+      };
+
+      constexpr std::int64_t row_count = 4;
+      constexpr std::int64_t column_count = 3;
+
+      auto shared_data = sycl::malloc_shared<float>(row_count * column_count, queue);
+      auto event = queue.memcpy(shared_data, host_data, sizeof(float) * row_count * column_count);
+      auto t = dal::homogen_table::wrap(queue, data, row_count, column_count, { event });
+
+      // Accessing second and third rows of the table
+      {
+         dal::row_accessor<const float> acc { t };
+         auto block = acc.pull(queue, {1, 3});
+         for(std::int64_t i = 0; i < block.get_count(); i++) {
+            std::cout << block[i] << ", ";
+         }
+         std::cout << std::endl;
+      }
+
+      // Accessing whole elements in a first column
+      {
+         dal::column_accessor<const float> acc { t };
+         auto block = acc.pull(queue, 0);
+         for(std::int64_t i = 0; i < block.get_count(); i++) {
+            std::cout << block[i] << ", ";
+         }
+         std::cout << std::endl;
+      }
+
+      sycl::free(shared_data, queue);
+      return 0;
+   }
 
 .. _row_accessor:
 
-Row accessor
-------------
+Row accessor programming interface
```

Comment:
is this aligned with other topics in spec?

I'd add a section named API, and then the subsections for row and column accessors interfaces:

-API
--Row accessor
--Column accessor